design.tex

```diff
@@ -158,64 +156,62 @@ \subsubsection{Clearinghouse}\label{sec-ch}
 registration form \yanyan{cite url or show screenshot} at the 
 clearinghouse. The clearinghouse website shows 
 a list of available sensors, and each of their required accuracy 
-and access frequency. Aftering filling out this form, the researcher 
+and access frequency.\lois(does it show the "sensors" apart from the devices?) After filling out this form, the researcher 
```

Comment:
No problem.


Lois

On Wed, Nov 11, 2015 at 05:42 PM, Yanyan Zhuang wrote:

In design.tex 
<https://github.com/yyzhuang/mobisys16-sensibility/pull/2#discussion_r44598015> 
:
> @@ -158,64 +156,62 @@ \subsubsection{Clearinghouse}\label{sec-ch}
>  registration form \yanyan{cite url or show screenshot} at the 
> clearinghouse. The clearinghouse website shows  a list of available 
> sensors, and each of their required accuracy -and access frequency. 
> Aftering filling out this form, the researcher +and access 
> frequency.\lois(does it show the "sensors" apart from the devices?) 
> After filling out this form, the researcher


I see what the confusion is now. Thanks!


—
Reply to this email directly or view it on GitHub 
<https://github.com/yyzhuang/mobisys16-sensibility/pull/2/files#r44598015> 
.